src/__test__/Utilities.test.ts

```diff
@@ -0,0 +1,279 @@
+      expect(error.debugMessage).toBe(testDebugMessage);
+    }
+  });
+
+  it('verifyIntegerType pass with valid integer type', () => {
+    const testDebugMessage = 'test debug message';
+    Utilities.verifyIntegerType(1, testDebugMessage);
+  });
+
+  it('verifyIntegerType create error with invalid type', () => {
+    expect.assertions(8);
+    const testDebugMessage = 'test debug message';
+    try {
+      Utilities.verifyIntegerType('string', testDebugMessage);
+    } catch (ex) {
+      const error:ErrorDetails = JSON.parse(ex.message);
+      expect(error.code).toBe(ErrorCodes.UsageError);
+      expect(error.message).toContain('rn_invalid_type');
+      expect(error.debugCode).toBe('rn_invalid_type');
+      expect(error.debugMessage).toBe(testDebugMessage);
+    }
+    try {
+      Utilities.verifyIntegerType(1.123, testDebugMessage);
+    } catch (ex) {
+      const error:ErrorDetails = JSON.parse(ex.message);
+      expect(error.code).toBe(ErrorCodes.UsageError);
+      expect(error.message).toContain('rn_invalid_type');
+      expect(error.debugCode).toBe('rn_invalid_type');
+      expect(error.debugMessage).toBe(testDebugMessage);
+    }
+  });
+
+  it('verifyBooleanType pass with valid boolean type', () => {
+    const testDebugMessage = 'test debug message';
+    Utilities.verifyBooleanType(false, testDebugMessage);
+  });
+
+  it('verifyBooleanType create error with invalid type', () => {
+    expect.assertions(4);
+    const testDebugMessage = 'test debug message';
+    try {
+      Utilities.verifyBooleanType(undefined, testDebugMessage);
+    } catch (ex) {
+      const error:ErrorDetails = JSON.parse(ex.message);
+      expect(error.code).toBe(ErrorCodes.UsageError);
+      expect(error.message).toContain('rn_invalid_type');
+      expect(error.debugCode).toBe('rn_invalid_type');
+      expect(error.debugMessage).toBe(testDebugMessage);
+    }
+  });
+
+  it('verifyNubmerType pass with valid number types', () => {
+    const testDebugMessage = 'test debug message';
+    Utilities.verifyNubmerType(123, testDebugMessage);
+    Utilities.verifyNubmerType(1.23, testDebugMessage);
+  });
+
+  it('verifyNubmerType create error with invalid type', () => {
+    expect.assertions(4);
+    const testDebugMessage = 'test debug message';
+    try {
+      Utilities.verifyNubmerType(true, testDebugMessage);
+    } catch (ex) {
+      const error:ErrorDetails = JSON.parse(ex.message);
+      expect(error.code).toBe(ErrorCodes.UsageError);
+      expect(error.message).toContain('rn_invalid_type');
+      expect(error.debugCode).toBe('rn_invalid_type');
+      expect(error.debugMessage).toBe(testDebugMessage);
+    }
+  });
+
+  it('verifyFontType pass with valid font types', () => {
+    Utilities.verifyFontType({
+      size: 14.3,
+      name: 'hei',
+    });
+    Utilities.verifyFontType({
+      size: 20,
+    });
+  });
+
+  // it('verifyFontType create error with invalid type', () => {
```

Comment:
Remove them or uncomment them?

Reply:
I will remove these. These test cases are now not feasible since we can not pass invalid parameters as they have now become now type-specific.